packages/atomic/src/components/search/result-template-components/atomic-quickview-sidebar/atomic-quickview-sidebar.tsx

```diff
@@ -86,47 +92,86 @@ const HighlightKeywordsCheckbox: FunctionalComponent<
           >
-            <div class="flex items-center grow p-4 border-r">
-              <div
-                class="w-5 h-5 flex-none mr-2"
-                style={{backgroundColor: keyword.color}}
-              ></div>
-              <div class="grow mr-2">{keyword.text}</div>
-              <div class="flex-none">{keyword.occurrences}</div>
-            </div>
-            <div class="flex px-2">
-              <atomic-icon-button
-                buttonStyle="text-transparent"
-                class="mr-2 border-0"
-                labelI18nKey="next"
-                tooltip={i18n.t('next')}
-                icon={ArrowDown}
-                clickCallback={() => {
-                  keyword.navigateForward();
-                }}
-              ></atomic-icon-button>
-              <atomic-icon-button
-                buttonStyle="text-transparent"
-                class="border-0"
-                labelI18nKey="previous"
-                tooltip={i18n.t('previous')}
-                icon={ArrowUp}
-                clickCallback={() => keyword.navigateBackward()}
-              ></atomic-icon-button>
+            <div
+              class={`flex grow items-center bg-background border border-neutral rounded-lg overflow-x-auto ${
+                !wordIsEnabled ? 'pointer-events-none opacity-50' : ''
+              }`}
+            >
+              <div class="flex items-center grow p-4 border-r">
+                <div
+                  class="w-5 h-5 flex-none mr-2"
+                  style={{backgroundColor: keyword.color}}
+                ></div>
+                <div class="grow mr-2">{keyword.text}</div>
+                <div class="flex-none">{keyword.occurrences}</div>
+              </div>
+              <div class="flex px-2">
+                <atomic-icon-button
+                  buttonStyle="text-transparent"
+                  class="border-0"
+                  labelI18nKey="next"
+                  tooltip={i18n.t('next')}
+                  icon={ArrowDown}
+                  clickCallback={() => {
+                    keyword.navigateForward();
+                  }}
+                ></atomic-icon-button>
+                <atomic-icon-button
+                  buttonStyle="text-transparent"
+                  class="border-0"
+                  labelI18nKey="previous"
+                  tooltip={i18n.t('previous')}
+                  icon={ArrowUp}
+                  clickCallback={() => keyword.navigateBackward()}
+                ></atomic-icon-button>
+              </div>
             </div>
+
+            <atomic-icon-button
+              class={`${
+                highlightKeywords.highlightAll
+                  ? ''
+                  : 'pointer-events-none opacity-50'
+              }`}
+              buttonStyle="text-transparent"
+              icon={wordIsEnabled ? Remove : Add}
+              labelI18nKey={
+                wordIsEnabled ? 'quickview-remove-word' : 'quickview-add-word'
```

Comment:
I prefer to avoid changing the `aria-label` of something that toggles, I recall getting accessibility issues about this a long time ago. You set the key to always be `quickview-add-word` and add `aria-pressed="true/false"` (not `undefined`, since a value of `false` will communicate to screen readers that this is an unpressed toggle button).